Pinta.Core/Actions/ImageActions.cs

```diff
@@ -265,16 +265,19 @@ private void HandlePintaCoreActionsImageAutoCropActivated (object sender, EventA
 
 			rect = new Gdk.Rectangle (rect.X, rect.Y, depth - rect.X, rect.Height);
 
-			CropImageToRectangle (doc, rect);
+			CropImageToRectangle (doc, rect, true);
 		}
 		#endregion
 
-		static void CropImageToRectangle (Document doc, Gdk.Rectangle rect)
+		static void CropImageToRectangle (Document doc, Gdk.Rectangle rect, bool isAutoCrop)
```

Comment:
Sorry for the delay in looking at these changes.

I think it would be cleaner to just have the caller pass in the history item string, rather than have a bool parameter.